yt_dlp/extractor/cbc.py

```diff
@@ -313,6 +314,37 @@ def _real_initialize(self):
             return
         self._claims_token = self._downloader.cache.load(self._NETRC_MACHINE, 'claims_token')
 
+    def _find_secret_formats(self, formats, video_id):
+        """ Find a valid video url and convert it to the secret variant """
+        base_format = next((f for f in formats if f.get('height')), None)
+        if not base_format:
+            return
```

Comment:
I don't understand why this is the selection criteria

Reply:
It needs to select a video, and width/height was the first differentiator that I found. Looking at it again, `if f.get('vcodec') != 'none'` could also work

Reply:
please do that. `acodec/vcodec='none'` is the canonical way to identify video/audio-only formats